### DISCUSSION TITLE
feat-4.3 cssvar support

### DISCUSSION THREAD

**Discussion by shifeng1993** [vueComponent/ant-design-vue#7939] [Show and tell]
同步版本 antd v5.22.1

# 目标分支
`feat-4.3`

# 进度
## 通用
- [ ] Button @shifeng1993 
- [ ] FloatButton
- [ ] Icon
- [ ] Typography

## 布局
- [ ] Divider
- [ ] Flex
- [ ] Row
- [ ] Col
- [ ] Layout
- [ ] Space
- [ ] Splitter

## 导航
- [x] Anchor @tangjinzhou 
- [ ] Breadcrumb
- [ ] Dropdown
- [ ] Menu @shifeng1993 
- [ ] Pagination
- [ ] Steps

## 数据录入
- [ ] AutoComplete
- [ ] Cascader
- [ ] Checkbox
- [ ] ColorPicker @shifeng1993 
- [ ] DatePicker
- [ ] Form
- [ ] Input
- [ ] InputNumber
- [ ] Mentions
- [ ] Radio
- [ ] Rate
- [ ] Select
- [ ] Slider
- [ ] Switch @shifeng1993 
- [ ] TimePicker
- [ ] Transfer
- [ ] TreeSelect
- [ ] Upload

## 数据展示
- [ ] Avatar
- [ ] Badge
- [ ] Calendar
- [ ] Card
- [ ] Carousel
- [ ] Collapse
- [ ] Descriptions
- [ ] Empty
- [ ] Image
- [ ] List
- [ ] Popover
- [ ] PageHeader
- [ ] QrCode
- [ ] Segmented
- [ ] Statistic
- [ ] Table
- [ ] Tabs
- [ ] Tag
- [ ] Timeline
- [ ] Tooltip
- [ ] Tour
- [ ] Tree

## 反馈
- [x] Alert @tangjinzhou 
- [ ] Drawer
- [ ] Message
- [ ] Modal
- [ ] Notification
- [ ] Popconfirm
- [ ] Progress
- [ ] Result
- [ ] Skeleton
- [ ] Spin
- [ ] WaterMark

## 其他
- [x] Affix @tangjinzhou 
- [ ] App
- [ ] ConfigProvider

**Comment by shifeng1993** [comment]
- Switch #7940

**Reply by wsxe9988** [reply]
加油，大佬

**Comment by JoeXu727** [comment]
- Spin

**Reply by wsxe9988** [reply]
多多贡献
